mods/ra/audio/notifications.yaml

```diff
@@ -113,7 +113,9 @@ Speech:
 		WarningFourMinutesRemaining: 4minr
 		WarningFiveMinutesRemaining: 5minr
 		Win: misnwon1
-
+	AltTexts:
```

Comment:
This was my ideal, but I chose simpler way for me. Notifications is Dictionary<string, string[]>so it can be BaseAttack: baseatk1, baseatk2. Adding new AltTexts dictionary just worked. I need to investigate more how to define and load it those additional properties.